package.json

```diff
@@ -39,7 +39,7 @@
     "xyz": "2.1.0"
   },
   "scripts": {
-    "lint": "eslint --ext .js,.ts .",
+    "lint": "eslint --ignore-path .gitignore --ignore-pattern /dist --ext .js,.ts .",
```

Comment:
And the other usage is here, the npm `lint` script.

Since `dist` is checked in, it is not ignored in `.gitignore` and has to be ignored here, specifically.

In the above VSCode usage, that is not a problem, because a file is not linted until it is opened. And ceases to be linted once it is closed. And the contents of `dist` *are* generated files, so they are rarely opened in the editor, anyway.